### ISSUE TITLE
Write a contribution guide

### ISSUE THREAD

**Issue by HEJOK254** [HEJOK254/Discord-QuickEdit#19]
I would like to contribute. Is there a guide on how to setup the project so I can test my changes?

_Originally posted by @BloodyMilchShake in https://github.com/HEJOK254/Discord-QuickEdit/issues/12#issuecomment-2241621045_
            
              For example the program wants a config file which apparently holds the api key but I think it would be helpful if there was an easy to read guide that would explain how the program is set up and what said config file expects.

_Originally posted by @BloodyMilchShake in https://github.com/HEJOK254/Discord-QuickEdit/issues/12#issuecomment-2241628266_
            

The guide should contain instructions on how to set up the project and contribute to it, as pointed out in the referenced comments above.
For help, visit: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

**Comment by BloodyMilchShake** [comment]
Thanks for your response. I'm familiar with cloning repositories and was referring to your specific project and how to setup and use it in it's current state. Like a sort of guide on how to get started with your project and how to use the bot in it's current state ^^

**Comment by HEJOK254** [comment]
> Thanks for your response. I'm familiar with cloning repositories and was referring to your specific project and how to setup and use it in it's current state. Like a sort of guide on how to get started with your project and how to use the bot in it's current state ^^

Thanks for the clarification

This issue is generally for the whole contribution guide, including the project specific setup, but also the general contribution info.

I replied to your previous comment with the instructions on how to set up the config file.\
Hope it helps! :)

**Comment by BloodyMilchShake** [comment]
Yeah sorry, I at first didn't realise this was an extra issue that aims at creating an contribution guide.
I tried deleting my comment after realising, but you appear to have replied too fast :P
